elements/lisk-codec/benchmark/encode_decode_biggest_lisk_transaction.js

```diff
@@ -0,0 +1,613 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+// writeBoolean x 3,543,238 ops/sec ±1.59% (89 runs sampled)
+
+const { Suite } = require('benchmark');
+const { codec } = require('../dist-node/codec');
+
+const suite = new Suite();
+
+const biggestMultisigTransactionRegistration = {
+	senderPublicKey: Buffer.from(
+		'0b211fce4b615083701cb8a8c99407e464b2f9aa4f367095322de1b77e5fcfbe',
+		'hex',
+	),
+	nonce: 1,
+	fee: BigInt(1500000000),
+	type: 12,
+	asset: {
+		mandatoryKeys: [
+			Buffer.from(
```

Comment:
maybe u can do this to avoid this `[ ...Array(64).keys() ].map(() => cr.buffcr.getRandomBytes(32))`

Reply:
Better to avoid random so input is always exactly the same? (it shouldn't matter but just to always keep exactly same input I would keep it fixed)

Reply:
true.. but since the length, type and input size remain constant, it shouldn't matter. just a suggestion for readability and less code to follow.